### PR TITLE
#99 Static info pages resource

### PR DESCRIPTION
**Pull request by sveredyuk**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -4,4 +4,8 @@ class ApplicationController < ActionController::Base
   helper_method :admin?
 
   delegate :admin?, to: :current_user, allow_nil: true
+
+  def not_found
+    render file: "#{Rails.root}/public/404", layout: false, status: :not_found
```
Comment (sveredyuk) [inline app/controllers/application_controller.rb:9]: @kushniryb @denysmedynskyi I am not sure that this is the best way to response with 404... what do you think guys ?

Comment (denys-medynskyi) [reply app/controllers/application_controller.rb:9]: I don't like it too, but I don't know how to do it better

**Review by denys-medynskyi — APPROVED** [review]

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (kushniryb) [reply app/controllers/application_controller.rb:9]: There's a dynamic approach, take a look
https://mattbrictson.com/dynamic-rails-error-pages

File: app/helpers/pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module PagesHelper
+  def page_link(page, options = {})
+    return unless page.persisted?
+
+    link_to (options[:text] || page.title),
```
Comment (kushniryb) [inline app/helpers/pages_helper.rb]: I guess `options.fetch(:text, page.title)` is more Rails way? :)

**Review by kushniryb — APPROVED** [review]
LGTM :+1:

**Pull request merged**